go/src/koding/db/mongodb/modelhelper/roles.go

```diff
@@ -130,3 +131,51 @@ func FetchAccountGroups(username string) ([]string, error) {
 
 	return slugList, nil
 }
+
+// FetchAccountGroups lists the groups of a given username
+func FetchAccountGroups(username string) (groups []*models.Group, err error) {
```

Comment:
you could share the part to fetch the ids of groups that we have in FetchAccountGroupNames